e2e/types.go

```diff
@@ -148,6 +148,10 @@ type LiveVMValidator struct {
 	// IsShellBuiltIn is a boolean flag which indicates whether or not the command is a shell built-in
 	// that will fail when executed with sudo - requires separate command to avoid command not found error on node
 	IsShellBuiltIn bool
+
+	// IsNonHostNetwork is a boolean flags which indicates whether or not the validator should run on a pod that is NOT using
+	// host's network interface. For example when testing connectivity from user pods to certain endpoints, we will set it to true
+	IsNonHostNetwork bool
```

Comment:
nit: `IsPodNetwork`

Reply:
honestly I don't think it makes sense to have a `LiveVMValidator` which is really a pod validator... I think we should have a new type called `PodValidator` or similar to distinguish between the 2 rather than overloading this one type

Reply:
we can leave as is and maybe add a TODO to separate pod validators from live VM validators